rust/src/types.rs

```diff
@@ -2263,4 +2338,14 @@ lazy_static! {
   };
   pub static ref FLOAT4X2_TYPES: Vec<Type> = vec![*FLOAT4X2_TYPE];
   pub static ref FLOAT4X2S_TYPE: Type = Type::seq(&FLOAT4X2_TYPES);
+  pub static ref TYPES_ENUM: Type = {
+    let mut t = common_type::enumeration;
+    t.details.enumeration = CBTypeInfo_Details_Enum {
+      vendorId: 0x66726167, // 'frag'
+      typeId: 0x74797065, // 'type'
+    };
+    t
+  };
+  pub static ref TYPES_ENUM_TYPES: Vec<Type> = vec![*TYPES_ENUM];
+  pub static ref TYPES_ENUMS: Type = Type::seq(&TYPES_ENUM_TYPES);
```

Comment:
**/!\\** I'm introducing an array of enum type in `ImGui_demo` branch (see 3aeb42d48424bfddc3e1ca2dcb689ee6229b8251).

Reply:
Sadly in rust for now they are re-defined :( hopefully one day we can fix